docs/features/automation_view.md

```diff
@@ -47,7 +47,8 @@ Automatable Parameters are broken down into three categories for Automation Clip
 
 3. Automatable CC's for MIDI Clips
 
->You can automate MIDI CC's 0-121, with 120 being Pitch Bend and 121 being After Touch
+>You can automate MIDI CC's 0-121, with 1 being Mod Wheel, 120 being Pitch Bend and 121 being After Touch
```

Comment:
CCs up to 119, along with aftertouch and pitch bend

Aftertouch and pitch bend aren't CCs so this is misleading otherwise